modules/fts/pages/fts-introduction.adoc

```diff
@@ -0,0 +1,124 @@
+
+= Introduction to Full Text Search
+
+== Full Text
+
+Provided by the xref:services-and-indexes/services/search-service.adoc[Search Service], this is a specially purposed index, which contains targets derived from the textual contents of documents within one or more specified keyspaces.
```

Comment:
`specially purposed` doesn't sound common in American English.  Also what are targets?

Reply:
Update: Remove "specially purposed" everywhere and just use "index"